readme.md

```diff
@@ -2,7 +2,27 @@
 
 Super-powered `laravel new` for Laravel and Valet.
 
-> NOTE: If you're working with the alpha, you'll need PHP 7.2, and this readme is currently out of date.The major change is that you'll want to run `lambo new myApplication` instead of `lambo myApplication`.
+>A new version of Lambo is coming that will introduce new features and provide a better platform for future enhancements... Oh, and it's being re-written from the ground-up in PHP! :fire: 
+>
+>If you are interested in checking out the beta, please read the *For beta testers* section below.
+
+# For beta testers
+
+To try the beta:
+
+```
+git https://github.com/tightenco/lambo.git lambo
```

Comment:
```suggestion
git clone https://github.com/tightenco/lambo.git lambo
```

Reply:
Could we just show them how to globally require the beta version instead?

Reply:
Potentially, yes. I will need to fix the Laravel Installer composer dependency as installation currently fails if the laravel 8 installer version is installed.